components/automate-es-gateway/habitat/plan.sh

```diff
@@ -10,11 +10,8 @@ pkg_maintainer="Chef Software Inc. <support@chef.io>"
 pkg_license=("Chef-MLSA")
 pkg_upstream_url="https://www.chef.io/automate"
 
-
-jdomain_version="1.1.5"
-jdomain_filename="ngx_upstream_jdomain-${jdomain_version}.tar.gz"
-jdomain_source="https://github.com/nicholaschiasson/ngx_upstream_jdomain/archive/${jdomain_version}.tar.gz"
-jdomain_shasum=e7346dac41d473ea02995b7626bd0660115538c2c784dfe683e235c58dbc2c2c
+jdomain_branch="kallol/set_max_tries_for_instances"
```

Comment:
Now that we've merged your change:

```suggestion
jdomain_branch="master"
```